tests/test_roles.py

```
@@ -301,11 +286,11 @@ def test_delete_role_with_reassignment(self):
                 pass
             except Exception as exp:
                 # Log warning but don't fail the test
-                print(f"Warning: Could not clean up user {self.user_id}: {exp}")
+                raise Exception(f"Warning: Could not clean up user {self.user_id}: {exp}")
```

Ruff (0.15.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)